src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -28,22 +62,34 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  @Test
+  public void testGetUserByID() {
```

Comment:
testGetUserByID(com.rtb.projectmanagementtool.user.UserControllerTest)  Time elapsed: 0.033 sec  <<< ERROR!
java.lang.NullPointerException
        at com.rtb.projectmanagementtool.user.UserData.<init>(UserData.java:48)
        at com.rtb.projectmanagementtool.user.UserController.getUserByID(UserController.java:19)
        at com.rtb.projectmanagementtool.user.UserControllerTest.testGetUserByID(UserControllerTest.java:76)

Reply:
I'm having trouble working through this Null Pointer Exception. Can't seem to piece together exactly what'd going on here. Any glaring error/suggestions of how to work through?

Reply:
I would print the userData/entity along each step (in the test classes, userController methods used, and userData methods used) and pay attention to what the id is. Maybe something isn't what you expect it to be?